extensions/resteasy-qute/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>quarkus-build-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../../build-parent/pom.xml</relativePath>
+    </parent>
+
+    <artifactId>quarkus-resteasy-qute-parent</artifactId>
+    <name>Quarkus - RESTEasy Qute</name>
```

Comment:
Minor but should be `Quarkus - RESTEasy - Qute` to follow the convention we have for Jackson and JSON-B. I can do it in a follow-up if you don't feel like doing it.

Same for the runtime and deployment artifacts obviously.

Reply:
What's the convention? The dash in the name?

Reply:
Ok, it's the dash. I misread your comment. Ok, done.